### ISSUE TITLE
[Ranks] Items added to Beast-Master Rank and issues with the ones we obtained those ranks before the changes

### ISSUE THREAD

**Issue by EzelBerbier** [NebulaMC-GG/Support#1576]
Hi, I know you've missed me, but unfortunately im here to give u more bad news. Yeah I know, nobody like it but it's necessary. 

*Dream and Luxury rank already solved, so i've edit this to delete the stuff about it cause not have sense and for don't make another ticket. But they have some issues after the fixes.

- Luxury rank gives u 16 ultra balls instead of 32.
- In the Dream Rank Rewards appears "1x 1x Master ball". Don't affect to the master ball itself, just visual bug.

**Specific issues**. I don't think this affect to the rest of the players or will affect the ones who reach those rank after this changes. Is a issue specific the ones who obtained the those before the changes, so I'm gonna put it on falling order starting with Master Rank. 

> [MASTER RANk]

|New changes added to the Master Rank rewards| The master rank have new items as a rewards and other changes that collides with the lastest rewards, first of all I will start with the items that aren't on the previous rewards:

- 3 master ball instead of 2. So, the ones that we already have master rank, we didn't receive that extra master ball. 
-  1 Ability Modifier
-  1 Dragon Skull 
-  32 Rare Candies instead 12. So we didn't receive those extra 20 rare candies.
- 512 claim blocks. Before was 812 claim blocks.

|Old rewards| Rewards that we obtained before, so I case the actually master ranks we wanted to redeem something, I think is fair to return them (thats Staff decision for sure). 

- 2 Specific Max Iv's modifier. Now is 1, so we obtained a extra one.
- 1 Strong HA lure. Deleted from the old reward. 

> [BEAST RANK]

|New changes added to the Beast Rank rewards|

- 3 Specific Max Ivs modifier instead of 1.
- 1 Elytra. it wasn't there before the changes.
- 1 Nature Modifier. it wasn't there before the changes.
- 16 Beast balls instead of 1. 
- 64 Quick Balls Instead of 30. 
- 24 thunderstones. it wasn't there before the changes.
- 2 Master balls instead of 1.
- 1 Tidal Bell. it wasn't there before the changes.
- 1 Clear ball. it wasn't there before the changes.
- 512 claim blocks instead of 812. 
- 5 XL Exp. Candies. Before that was 10.

|Old rewards|

- 1 Amulet Coin. Deleted now.
- 1 Soothe Bell. Deleted now.
- 1 Netherite Pickaxe. Deteled now.
- 1 Destiny Knot. Deleted now.
- 1 Infuser. Deleted now.
- 24 Ultra Balls. Deleted now.

I reported this just for one reason: the new changes added to Beast and Master Rank gives u extra items more expensive than the previous one, specifically for Beast Rank, let us the one who redeem the old one with a worse reward, and being a one-time rewards is not fair, i mean, the new beast ranks will obtain a free lugia and ho-oh wih the bells (for example), why can't obtian it too? So this is my report, to fix this, cause being master rank obviously i want to obtain those items. I don't think some items like the balls (except mballs) are really important as a return, but other items more hard to find or with a really economic value are really important.

To solve this as better as possible, I will create a new comment here with the items (In case the staff redeem it to the beast and master ranks) with the new items added, so that will help to do it faster without thinking about it, cause this is hella confuse it sometimes. 
 
In case is needed, i'm miguipkv, player of Atlas. My discord Id is Ezel#1229 in case someone of the staff needs to contact me in private for help or clarify something about all this chaos.

**Comment by EzelBerbier** [comment]
Now I will proceed with the second part, more focused in the possible ways to fix this (or the ways that i think could fix this), for the ones that we were beast and master rank before the changes and we can't obtain those items due to is one-time reward, and make the changes fair for all of us. The decision, whatever will end its a staff decision, for sure. 

Due to those changes, i will make a list, in case the beast and master rank want to report it to obtain those new items added to the rewards, with the items tha the staff should give them who the ones that redeemed the rank before the changes. 

> [Master Rank] 

List of the items that the staff should give to the master ranks who redeem the reward before the changes:

- 1 Masterball
- 1 Ability Modifier
- 1 Dragon Skull
- 20 Rare Candies

Items that the player should return in case the staff ask for them:

- 1 Specific Max Iv's modifier
- 1 Strong HA lure. 
- 300 claim blocks (not sure if thats possible, i think a admin should do it manually).

> [Beast Rank]

List of the items that the staff should give to the master ranks who redeem the reward before the changes:

- 2 Specific Max Ivs modifier 
- 1 Elytra
- 1 Nature Modifier. 
- 15 Beast balls instead 
- 34 Quick Balls 
- 24 thunderstones
- 1 master ball
- 1 Tidal Bell. 
- 1 Clear ball. 

Items that the player should return in case the staff ask for them:

- 1 Amulet Coin. 
- 1 Soothe Bell.
- 1 Netherite Pickaxe
- 1 Destiny Knot
- 1 Infuser
- 24 Ultra Balls

The beast reward should be redeem to for the master ranks, not only for beast ball, cause affected them too. Exist other solution, easier than this but less fair, that is let the beast and master ranks that redeemd it again, bein less fair cause they will win extra items and stuff, but its easier for the statt to fix the issue. 

The final solution its staff decision, like i said. Hope my reports are helping to fix this as far as possible cause i really want those new items added, and i think I wont be the only one. Good luck with all, great work with the ranks even with the issues, but well, beta versions are always a mess.

**Comment by EzelBerbier** [comment]
Hi, I'm here for Ras petition to put the date that I obtained Beast and Master Rank due to items changed issue. 

I obtained Beast Rank 27/07/2022 and Master Rank 30/07/2022

**Comment by EzelBerbier** [comment]
Now I will proceed with the second part, more focused in the possible ways to fix this (or the ways that i think could fix this), for the ones that we were beast and master rank before the changes and we can't obtain those items due to is one-time reward, and make the changes fair for all of us. The decision, whatever will end its a staff decision, for sure.

Now I will proceed with the list of the items added and the ones should be returned, as apart of the first solution that i thought:

[Master Rank]

List of the items that the staff should give to the master ranks who redeem the reward before the changes:

1 Masterball
1 Ability Modifier
1 Dragon Skull
20 Rare Candies
Items that the player should return in case the staff ask for them:

1 Specific Max Iv's modifier
1 Strong HA lure.
300 claim blocks (not sure if thats possible, i think a admin should do it manually).
[Beast Rank]

List of the items that the staff should give to the master ranks who redeem the reward before the changes:

2 Specific Max Ivs modifier
1 Elytra
1 Nature Modifier.
15 Beast balls instead
34 Quick Balls
24 thunderstones
1 master ball
1 Tidal Bell.
1 Clear ball.
Items that the player should return in case the staff ask for them:

1 Amulet Coin.
1 Soothe Bell.
1 Netherite Pickaxe
1 Destiny Knot
1 Infuser
24 Ultra Balls

The final solution its staff decision, like i said. Hope my reports are helping to fix this as far as possible cause i really want those new items added, and i think I wont be the only one. Good luck with all, great work with the ranks even with the issues, but well, beta versions are always a mess.

Hi, I'm here for Ras petition to put the date that I obtained Beast and Master Rank due to items changed issue.

I obtained Beast Rank 27/07/2022 and Master Rank 30/07/2022
